src/phases/execution/TimeSeriesSortCommands.yml

```diff
@@ -0,0 +1,30 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  These are the phases used to measure performance of the bounded sorter for timeseries collections.
+
+BlockingSortCmdTemplate:
+  aggregate: {^Parameter: {Name: "coll", Default: Collection0}}
+  pipeline: [{$sort: {t: 1}}, {$_internalInhibitOptimization: {}}, {$skip: 1e10}]
```

Comment:
Ah sorry these BUS perf tickets are kind of confusing because in the initial PR, I put the blocking and bounded sort tests, but then when we merge the ticket I remove the blocking sort tests since we only want to merge tests related to the BUS work. I'll change this to WIP to make it more clear.

In the future this will be optimized to a bounded sort but since we're comparing blocking and bounded sort it should be fine (but also adding the $_internalInhibitOptimization doesn't hurt)

Reply:
Oh sorry actually to be clear, before merging I delete the manually written bounded sorter pipelines, because the pipelines with $sort will eventually be optimized to bounded sort.